### PR TITLE
Add feedback from PR.

### PR DESCRIPTION
**Pull request by NaadiQmmr**

### CODE REVIEW

File: security/ransomware/README.md [hunk 1]
```diff
@@ -6,40 +6,19 @@ With this Workshop, you'll learn:
...
 
-This Workshop is for educational purposes only. Poc Innovation isn't responsable
+:warning: This Workshop is for educational purposes only. Poc Innovation isn't responsable
 for what you will do with this knowledge.
 
-We provided you a `Document/` directory to infect. Be careful not to infect your own and important files.
+We provided you a `Documents/` directory to infect. Be careful not to infect your own and important files.
 If you are afraid to do so, please do this workshop on a virtual machine.
 
-# Installation
```
Comment (TomChv) [inline security/ransomware/README.md:15]: Add a line to add the setup like [this](https://github.com/PoCInnovation/Workshops/tree/master/software/14.Vercel#step-0---setup)

File: security/ransomware/README.md [hunk 2]
```diff
@@ -105,6 +92,16 @@ You can:
 
 Congratulations !
 
+# Go further
```
Comment (TomChv) [inline security/ransomware/README.md:95]:
```suggestion
## Go further
```

File: security/ransomware/README.md [hunk 3]
```diff
@@ -105,6 +92,16 @@ You can:
...
+
+You can :
+* not only encrypt files, but also dump the whole computer that you infects, and get some user informations.
+* implement a web server on your code, that will send the data files.
+* on this web server, you can develop some stats about how many people did (fakely) pay, how many files did you infects, etc.
+
+# Some usefuls links
```
Comment (TomChv) [inline security/ransomware/README.md:102]:
```suggestion
## More resources
```

File: security/ransomware/README.md [hunk 4]
```diff
@@ -105,6 +92,16 @@ You can:
...
+[Great GUI interface](https://github.com/leonv024/RAASNet)
+
 # Authors
```
Comment (TomChv) [inline security/ransomware/README.md:105]:
```suggestion
## Authors
```

File: security/ransomware/README.md [hunk 5]
```diff
@@ -55,11 +34,19 @@ Fill the functions in sources/encryption.c.
...
+
+```sh
+make ; ./ransom -d [folder_name] [password]
+```
 
-You can learn more about it [here](https://en.wikipedia.org/wiki/Obfuscation_(software))
+Some functions of the libsodium may afraid you with their long and weird names.
+Read [this](https://github.com/jedisct1/libsodium-doc/blob/master/secret-key_cryptography/secretstream.md) to more informations.
+
+# Obfuscation
```
Comment (TomChv) [inline security/ransomware/README.md:47]:
```suggestion
## Obfuscation
```

File: security/ransomware/README.md [hunk 6]
```diff
@@ -6,40 +6,19 @@ With this Workshop, you'll learn:
...
-sudo apt-get install -y libsodium-dev
-```
-
-## Fedora
-
-```sh
-sudo dnf install -y libsodium-devl
-```
 
 # Encryption
```
Comment (TomChv) [inline security/ransomware/README.md:16]:
```suggestion
## Encryption
```

File: security/ransomware/README.md [hunk 7]
```diff
@@ -6,40 +6,19 @@ With this Workshop, you'll learn:
 
 # Disclaimer
```
Comment (TomChv) [inline security/ransomware/README.md:7]:
```suggestion
## Disclaimer
```

File: security/ransomware/SETUP.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+```sh
+sudo apt-get update -y
+sudo apt-get install -y libsodium-dev
+```
+
+## Fedora
+
+```sh
+sudo dnf install -y libsodium-devel
+```
```
Comment (TomChv) [inline security/ransomware/SETUP.md:16]:
```suggestion
```

Go back to the [exercise](./README.md)
```

**Review by TomChv — CHANGES_REQUESTED** [review]
Great Job, minor style change to do

File: security/ransomware/SETUP.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+Go back to the [exercise](./README.md)
```
Comment (NaadiQmmr) [inline security/ransomware/SETUP.md:18]: @TomChv  done here

File: security/ransomware/README.md [hunk 10]
```diff
@@ -4,42 +4,23 @@ With this Workshop, you'll learn:
...
 You must write an encryption program compliant with the decryption already provided,
 otherwise, we won't be able to help you.
 
-### How to run decryption ?
-
-```sh
-make ; ./ransom -d [folder_name] [password]
-```
+### Step 0 - Setup
+All the required informations can be found in the [SETUP.md](./SETUP.md)
```
Comment (NaadiQmmr) [inline security/ransomware/README.md:24]: @TomChv and done here !

**Review by TomChv — APPROVED** [review]
LGTM

**Pull request merged**